client/components/cart.jsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-shadow */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import { getCartItems } from '../redux/cart/actions';
+import { orderParser } from '../utilities';
+import { getMovies } from '../redux/movies/actions';
+
+class Cart extends Component {
+  async componentDidMount() {
+    const { getCartItems, getMovies } = this.props;
+
+    await getMovies();
+    await getCartItems();
+  }
+
+  render() {
+    const { orders, total, movies } = this.props;
+    console.log('movies ', movies);
```

Comment:
If these console.logs are no longer being used for debugging, it'd be good to delete them!